### PR TITLE
integrate with backend email verify api.

### PR DESCRIPTION
**Pull request by fivestarwy**

### CODE REVIEW

File: src/shared/components/Settings/Account/MyAccount/EmailVerifiResult/Success/index.jsx [hunk 1]
```diff
@@ -1,39 +1,73 @@
...
+    render() {
+        let assets;
+        if (isomorphy.isClientSide()) {
+            assets = require.context('assets/images/account/email', false, /svg/);
+        }
+        const { isLoading } = this.state;
+        if (isLoading) {
+            return  <LoadingPagePlaceholder />;
+        }
+        const {handle,token,newEmail,oldEmail,jwtToken} = this.props.match.params;
```
Comment (dedywahyudi) [inline src/shared/components/Settings/Account/MyAccount/EmailVerifiResult/Success/index.jsx]: @fivestarwy This is declaraed twice? See line 23

**Comment by dedywahyudi** [comment]
@fivestarwy It look we need clean up this commit more to follow community-app structure: 
- const {
      handle,
      token,
      newEmail,
      oldEmail,
      jwtToken,
    } = this.props.match.params;
- .catch(err => this.setState({ statusCode: 400, isLoading: false })), 2000);

**Comment by dedywahyudi** [comment]
Hi @birdofpreyru @ThomasKranitsas, 

Can you help check how to solve these lint error please? 
Any idea how to solve these **match.params** prop-type issue? 
fyi: that label-has-for issues not coming from our updates. 

https://circleci.com/gh/topcoder-platform/community-app/2135?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

screenshot 2018-07-16 20 30 59

**Comment by birdofpreyru** [comment]
And, probably, match and params should required props.

**Comment by dedywahyudi** [comment]
That's ultrafast respond, thanks guys!

**Pull request merged**